packages/react-next/src/components/SwatchColorPicker/SwatchColorPicker.base.tsx

```diff
@@ -16,361 +8,316 @@ import {
 import { Grid } from '../../utilities/grid/Grid';
 import { IColorCellProps } from './ColorPickerGridCell.types';
 import { ColorPickerGridCell } from './ColorPickerGridCell';
-import { memoizeFunction, warnDeprecations } from '@uifabric/utilities';
-
-export interface ISwatchColorPickerState {
-  selectedIndex?: number;
+import { useId, useConst, useSetTimeout, useControllableValue, useWarnings } from '@uifabric/react-hooks';
+import { IGridProps } from '../../utilities/grid/Grid.types';
+
+interface ISwatchColorPickerState {
```

Comment:
Nitpicky thing but it would be better to call this `ISwatchColorPickerInternalState` to reduce potential confusion with the old version of React state.